suite/tests/CMakeLists.txt

```diff
@@ -2223,6 +2223,7 @@ if (CLIENT_INTERFACE)
   use_DynamoRIO_extension(client.drxmgr-test.dll drx)
   use_DynamoRIO_extension(client.drxmgr-test.dll drreg)
 
+  /* Declare reset params due to i#1674 */
   tobuild_ci(client.low_on_memory client-interface/low_on_memory.c ""
       "-enable_reset -reset_at_vmm_percent_free_limit 10 -vm_size 4M" "")
```

Comment:
I expected an explicit pointer to #3912 here for the  -reset_at_vmm_percent_free_limit 10.  Then someone fixing that can search the code and easily find where to update.

Reply:
Okay, I'll include that pointer as well. Thank you for taking the time to review the PR

Reply:
Done. I left the pointer to #1674 due to the enable-reset param.